From: Jiang Xin <jiangxin@ossxp.com>
Subject: [PATCH] t/config_new_defaults

<patch description>

Signed-off-by: Jiang Xin <jiangxin@ossxp.com>

---
 conf/example.gitolite.rc |    4 ++--
 1 个文件被修改，插入 2 行(+)，删除 2 行(-)

diff --git a/conf/example.gitolite.rc b/conf/example.gitolite.rc
index 6ffaa87..cc347e1 100644
--- a/conf/example.gitolite.rc
+++ b/conf/example.gitolite.rc
@@ -25,11 +25,11 @@ $GL_PACKAGE_HOOKS = "/tmp/share/gitolite/hooks";
 # ------------------------------------------------------------------------------
 # most often used/changed variables
 # ------------------------------------------------------------------------------
-$GL_WILDREPOS = 0;
+$GL_WILDREPOS = 1;
 $PROJECTS_LIST = $ENV{HOME} . "/projects.list";
 # $WEB_INTERFACE = "gitweb";
 # $GITWEB_URI_ESCAPE = 0;
-$REPO_UMASK = 0077;
+$REPO_UMASK = 0007;
 
 # ------------------------------------------------------------------------------
 # variables with an efficiency/performance impact
-- 
tg: (d59bc35..) t/config_new_defaults (depends on: upstream)
